src/relay/transforms/pattern_util.h

```diff
@@ -398,7 +398,8 @@ static inline long double ToScalar(const runtime::NDArray& array, size_t i = 0)
   } else if (array->dtype.code == kDLFloat) {
 #if (__ARM_FP16_FORMAT_IEEE == 1)
```

Comment:
perhaps we no longer need this macro guard anymore

Reply:
Right, removed.